doc/source/whatsnew/v1.5.0.rst

```diff
@@ -82,32 +82,40 @@ did not have the same index as the input.
 
 .. code-block:: ipython
 
+    In [3]: df.groupby('a', dropna=True).transform('sum')
+    Out[3]:
+       b
+    0  5
+    1  5
+    2  5
+
     In [3]: df.groupby('a', dropna=True).transform(lambda x: x.sum())
     Out[3]:
        b
     0  5
     1  5
 
+    In [3]: df.groupby('a', dropna=True).transform('ffill')
```

Comment:
add a comment here on the casting of the fill value

Reply:
I think you're saying to explain why the old result comes out as `-9223372036854775808`, which is `np.nan` when interpreted as an integer. Will do.